### DISCUSSION TITLE
Questions Regarding PHOEBE: Semi-Detached Binary Systems and Plot Customization

### DISCUSSION THREAD

**Discussion by mousam100** [phoebe-project/phoebe2#925] [Questions]
Hello Developers,

I have two questions regarding PHOEBE and its plotting:

Is it possible to simulate a binary system with Semi-Detached parameters and the S.D filter that undergoes mass transfer, transitioning from a Semi-Detached state to a Contact state? 

How can I change the axis label sizes and the xtick and ytick sizes in PHOEBE mesh plots?
''''''
IN THIS CODE:
%matplotlib inline
afig, mplfig = b['mesh01@model'].plot(x='ws', show=True)
mplfig.savefig('mesh1.png')

""""

Thank you for your assistance.

**Comment by kecnry** [comment]
> Is it possible to simulate a binary system with Semi-Detached parameters and the S.D filter that undergoes mass transfer, transitioning from a Semi-Detached state to a Contact state?

Unfortunately, no, systems must either be always or never in contact and each component has a fixed mass (in time).

> How can I change the axis label sizes and the xtick and ytick sizes in PHOEBE mesh plots?

You can modify the `mplfig` (which is a matplotlib figure object) using any matplotlib settings/commands between phoebe and saving it.  See #906 for a similar discussion (but for corner plots from distributions).

Hope this helps!

**Reply by mousam100** [reply]
Thanks Kyle.
